### DISCUSSION TITLE
Yardım dağıtılan yerlerin duyurulması

### DISCUSSION THREAD

**Discussion by alaradirik** [acikkaynak/deprem-yardim-projesi#63] [Ideas]
Merhaba, çok kişiden nerelerde yardım yapıldığının bilinmediğini ve yardımların efektif ulaştırılamadığını duyuyorum. deprem.io veya depremyardım.com'a böyle bir özellik ekleyebilir miyiz? Yardım adresi bildirme formu gibi yardım yapılan adres bildirme formu veya canlı harita yapılabilir.

**Comment by ramazansancar** [comment]
Merhaba,
Öncelikle fikriniz için teşekkürler. Bu talebiniz ile ilgili olarak [deprem-io](https://github.com/acikkaynak/deprem-io) için issues veya pr açabilirsiniz.
